### PR TITLE
Update parameter-types.md

### PR DESCRIPTION
**Pull request by achabanStalker**

### CODE REVIEW

**Review by DeadByElpy — DISMISSED** [review]

File: docs/analog/parameter-types.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ Command body structures for [SetParameter](./commands/SetParameter.md).
 * [Events config](#events-config)
 * [NBIOT module info](#NBIOT-module-info)
 * [NBIOT bands](#NBIOT-bands)
+* [NBIOT apn](#NBIOT-apn)
```
Comment (DarkPark) [inline docs/analog/parameter-types.md:43]:
```suggestion
* [NB-IoT APN](#nbiot-apn)
```

and in other places

seems `NB-IoT` is a standardized form

File: docs/analog/parameter-types.md [hunk 2]
```diff
@@ -1556,3 +1557,37 @@ Currently, preferred NB-IoT bands to be searched for. if set to 0 all bands that
 
 Message hex dump no LRC: `03 05 34 01 14`
 
+
+## NBIOT APN
+
+Parameter to set default APN.
```
Comment (DarkPark) [inline docs/analog/parameter-types.md:1563]:
```suggestion
Parameter to set the default APN.
```

File: docs/analog/parameter-types.md [hunk 3]
```diff
@@ -1556,3 +1557,37 @@ Currently, preferred NB-IoT bands to be searched for. if set to 0 all bands that
...
+### Format
+
+| Size   | Type     | Field                                   |
+| ----   | -------  | --------------------------------------- |
+| `1`    | `uint8`  | parameter type = `53`                   |
+| `1-64` | `string` | [apn](#apn)                             |
+
+#### **apn**
+A logical name that is used to select the GGSN or the external packet data 
+network. If the string size is 0 then the apn will be "network provided APN".
```
Comment (DarkPark) [inline docs/analog/parameter-types.md:1578]:
```suggestion
network. If the string size is `0` then the APN will be "network provided APN".
```
?

File: docs/analog/parameter-types.md [hunk 4]
```diff
@@ -1556,3 +1557,37 @@ Currently, preferred NB-IoT bands to be searched for. if set to 0 all bands that
...
+Parameter to set default APN.
+Available from software version = `1.5` for:<br/>
+hardware type - `24`
+
+[Hardware types](./basics.md#hardware-types)
+
+### Format
+
+| Size   | Type     | Field                                   |
+| ----   | -------  | --------------------------------------- |
```
Comment (DarkPark) [inline docs/analog/parameter-types.md:1572]:
```suggestion
| ------ | -------- | --------------------------------------- |
```

but it's best to use autoformatter

**Review by DarkPark — APPROVED** [review]

**Pull request merged**